src/styles/app.css

```diff
@@ -137,7 +137,7 @@ h6 {
     outline: none;
 }
 
-.MuiPopper-root {
+[data-cy="testFloatingLinkToolbar"] {
```

Comment:
suggestion (non-blocking): we shouldn't identify the element in CSS with the cypress identificator.